scripts/mkcloud

```diff
@@ -101,7 +112,7 @@ needcvol=1
 : ${xen_node_memory:=4000000}
 [[ "$libvirt_type" = "xen" && $compute_node_memory -lt $xen_node_memory ]] && compute_node_memory=$xen_node_memory
 # hdd size defaults (unless defined otherwise)
-: ${adminnode_hdd_size:=15}
+: ${adminnode_hdd_size:=20}
```

Comment:
If you need more for a special deployment, then just set this variable from the outside. I don't see a reason to increase the default size.